### DISCUSSION TITLE
Custom Domain UNDO

### DISCUSSION THREAD

**Discussion by rgeorgia** [community/community#76946] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I wanted to set my github.io to a custom domain. I entered the domain name and clicked "save". I guess I did not understand some of the nuances for configuring; consequently I messed things up. I removed the entry; however, my github.io is still being redirected to the custom domain. Is there any way I can restore the DNS to point back to my github.io?

**Comment by smouad** [comment]
1- Remove Custom Domain in GitHub:
        Go to your GitHub repository.
        Click on "Settings" in the right sidebar.
        Scroll down to the "GitHub Pages" section.
        In the "Custom domain" box, remove the custom domain and click "Save."

    2- Update DNS Records:
        Go to your domain registrar's website (where you purchased the domain).
        Find the DNS settings or DNS management section.
        Remove any custom DNS records related to your GitHub Pages.

    This step may take some time as DNS changes propagate across the internet. Be patient, and your GitHub Pages should revert to the default github.io domain.

If the issue persists, consider clearing your browser cache or trying from a different browser. If it still doesn't work, there might be a caching issue that will resolve itself over time.
